websites/documentation/src/content/index.mdx

```diff
@@ -2,6 +2,14 @@
 title: The commercetools Docs-Kit.
 ---
 
+import { LoginButton, IfUserLoggedOut, IfUserLoggedIn } from '@commercetools-docs/gatsby-theme-docs'
+
+<div style="display:flex;">
+
+  <LoginButton theme="primary" label="Log in"/>
+
+</div>
+
```

Comment:
Looks rather like a smoke test than a part of the kit documentation

Reply:
true .. I was just testing with that .. I'll change it